### ISSUE TITLE
Needs root permission to make new app under Mac OS High Sierra

### ISSUE THREAD

**Issue by zaffka** [gobuffalo/buffalo#695]
> MacBook-Pro-az:src az$ buffalo new buffalo_test
> Buffalo version v0.9.5
> 
> --> go get -u golang.org/x/tools/cmd/goimports
> --> go get -u github.com/golang/dep/cmd/dep
> \# cd /Users/az/go/src/github.com/golang/dep; git pull --ff-only
> error: cannot open .git/FETCH_HEAD: Permission denied
> package github.com/golang/dep/cmd/dep: exit status 1

So I forced to launch it under sudo, and make 'sudo chown -R {username} {buffalodir}' after.

**Comment by markbates** [comment]
It shouldn't need root permission. It doesn't look like your go installation isn't correct. In particular it looks like the `/Users/az/go/src/github.com/golang/dep` has strange permissions on it. Check the permissions of the `$GOPATH` and make sure those are correct. You should never need root to do a `go get`.

**Comment by zaffka** [comment]
To test buffalo I installed node.js 6.11.4 with npm using official package for mac.
After that I tried `buffalo new` and it failed to install yarn so I gave it root permission.
After that process was finished successfuly, but something (I really think it was yarn) started to work as from 'root' user.

Now I've reinstalled everything (node and npm using brew) and buffalo works just fine 👍 

Thank you so much for your job, Mark!
Sorry for such baby complain - i'm a novice in programming, hope somehow this information helps to tune the processes.
